src/config.c

```diff
@@ -355,6 +355,22 @@ struct vifconfig *parsePhyintToken(void) {
                 my_log(LOG_WARNING, 0, "Unable to parse subnet address.");
                 break;
             } else {
+                (*agrpPtr)->allow = 1;
+                agrpPtr = &(*agrpPtr)->next;
+            }
+        }
+        else if(strcmp("blacklist", token)==0) {
+            // Blacklist
+            token = nextConfigToken();
+            my_log(LOG_DEBUG, 0, "Config: IF: Got blacklist token %s.", token);
+
+            *agrpPtr = parseSubnetAddress(token);
+            if(*agrpPtr == NULL) {
+                parseError = 1;
+                my_log(LOG_WARNING, 0, "Unable to parse subnet address.");
```

Comment:
This should be (fatal) error. When config line is invalid igmpproxy should not try to ignore or change keyword to something else. (Same applies for `whitelist` keyword parsing).

Reply:
`blacklist` and `whitelist` now give an error when parsing fails.